libraries/mbed/hal/can_api.h

```diff
@@ -37,7 +37,8 @@ typedef enum {
     IRQ_PASSIVE,
     IRQ_ARB,
     IRQ_BUS,
-    IRQ_READY
+    IRQ_READY,
+    IRQ_RX1
```

Comment:
Hello, I just comitted this file again, explaining the new interrupt in the commit message.
There are 2 fifos in CAN. Each fifo generates its own interrupt in case of reception / fifo-full or overrun.
Adding a new value to the enum should have no effect on other microcontrollers.